process/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,134 @@
+          duration in seconds specified by access_denied_cache_duration.
+        value:
+          type: integer
+          example: 120
+      - name: shared_process_list_cache_duration
+        description: |
+          The check maintains a list of running processes shared among all instances, that is used to generate the
+          matching pid list on each instance. It won't try to look at them again for the duration in seconds
+          specified by shared_process_list_cache_duration.
+        value:
+          type: integer
+          example: 120
+      - name: procfs_path
+        description: |
+          Used to override the default procfs path, e.g. for docker containers with the outside fs mounted at /host/proc
+          DEPRECATED: please specify `procfs_path` globally in `datadog.conf` instead
+        value:
+          type: string
+          example: /proc
+          default: null
+      - template: init_config/default
+    - template: instances
+      options:
+      - name: name
+        description: Used to uniquely identify your metrics as they are tagged with this name in Datadog.
+        required: true
+        value:
+          type: string
+      - name: search_string
+        description: |
+          If one of the elements in the list matches, it return the count of
+          all the processes that match the string exactly by default. Change this behavior with the
+          parameter `exact_match: false`.
+
+          Note: One and only one of search_string, pid or pid_file must be specified per instance.
+        value:
+          type: array
+          items:
+            type: string
+          exampe:
+            - '<SEARCH_STRING_1>'
+            - '<SEARCH_STRING_2>'
+      - name: pid
+        description: |
+          A Process id to match.
+
+          Note: One and only one of search_string, pid or pid_file must be specified per instance.
+        type:
+          value: integer
+      - name: pid_file
+        description: |
+          A Pid file to match.
+          Notes:
+            * agent v6.11+ on windows runs as an unprivileged `ddagentuser`, so make sure to grant it read access to
+              the PID files you specify in this option.
+            * One and only one of search_string, pid or pid_file must be specified per instance.
+        value:
+          type: string
+      - name: exact_match
+        description: |
+          Matches your search_string on proc.name().
+          If you want to match on a substring within proc.cmdline(), set this to false
+          Regex is also supported when this flag is set to `false`.
+
+          Note: agent v6.11+ on windows runs as an unprivileged `ddagentuser` that does not have acces to the full
+          command line of processes running under a different user. This option cannot be used in such cases.
+          https://docs.datadoghq.com/integrations/process/#configuration
+        value:
+          type: boolean
+          default: true
```

Comment:
```suggestion
```

Reply:
it complains that it needs default values otherwise

Reply:
Booleans must set `example`. Try again?